extensions/amp-render/1.0/test/test-amp-render.js

```diff
@@ -208,10 +208,16 @@ describes.realWin(
       doc.body.appendChild(element);
 
       await getRenderedData();
+      let options = fetchJsonStub.getCall(0).args[2];
+      // verify initial call to `batchFetJsonFor`
+      expect(options.refresh).to.be.false;
```

Comment:
Suggestion: [`match`](https://sinonjs.org/releases/latest/matchers/) on the full `options` object to also assert values for `expr` and `xssiPrefix` (even if trivial, it can be good to make sure this is in fact the case)